lua/HUDManagerPD2.lua

```diff
@@ -280,8 +280,8 @@ function HUDManager.show_interact(self, data)
 	if self._interact_visible and not data.force then
 		return
 	end
-
-	if SydneyHUD:GetOption("push_to_interact") and managers.player:toolset_value() >= SydneyHUD:GetOption("push_to_interact_delay")  then
+	local timer_data = 0 * managers.player:toolset_value()
+	if SydneyHUD:GetOption("push_to_interact") and timer_data >= SydneyHUD:GetOption("push_to_interact_delay")  then
```

Comment:
[L283](https://github.com/SydneyMOD/SydneyHUD/pull/53/files#diff-209924203408a9ab73c13c62921883ebR283) always return 0 I think,
[L284](https://github.com/SydneyMOD/SydneyHUD/pull/53/files#diff-209924203408a9ab73c13c62921883ebR284) can re-write to `if SydneyHUD:GetOption("push_to_interact") and 0`?